File: cfgs/java_cfgs_foo/POSITION_ELEMENT_STABLE_SORT.java
Nodes
1;special;;;
2;statement;int result = 0;14;
3;statement;int i = 0;15;
4;conditional;foo_1 < foo_2;16;
5;conditional;foo_1[foo_2] < foo_3[foo_4];18;
6;statement;result++;18;
7;conditional;(foo_1[foo_2] == foo_3[foo_4]) && (foo_5 < foo_6);19;
8;statement;result++;19;
9;statement;i++;17;
10;statement;return foo_1;21;
11;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;10;
4;6;7;
5;7;
6;8;9;
7;9;
8;4;
9;11;
10;

File: cfgs/pythoncfgs/POSITION_ELEMENT_STABLE_SORT.py
Nodes
1;statement;foo_1 = 0;8;
2;conditional;fun_1(foo_1);9;
3;statement;return foo_1;14;
4;conditional;foo_1[foo_2] < foo_3[foo_4];10;
5;conditional;foo_1[foo_2] == foo_3[foo_4] and foo_5 < foo_6;12;
6;statement;foo_1 += 1;13;
7;statement;foo_1 += 1;11;
8;exit;;
Edges
0;2;
1;3;4;
2;8;
3;5;7;
4;2;6;
5;2;
6;5;
7;
